marklogic-data-hub/src/main/java/com/marklogic/hub/flow/impl/FlowRunnerImpl.java

```diff
@@ -657,17 +657,32 @@ protected void applyArtifactSettings(Step step, Map<String, Object> stepConfig)
                     if (artifactSettingsJson.hasNonNull("customHook")) {
                         step.setCustomHook(artifactSettingsJson.get("customHook"));
                     }
-                    List<String> stepOptionProperties = Arrays.asList("collections", "permissions");
                     ObjectMapper mapper = new ObjectMapper();
+                    List<String> collections = new ArrayList<>();
```

Comment:
I think verifyThatLoadDataSettingsAreAppliedToStepConfig in FlowRunnerTest should be expanded to cover these fixes.

Reply:
done